core/platform/cache/redis_cache_services.py

```diff
@@ -25,15 +25,21 @@
 from typing import Dict, List, Optional
 
 # Redis client for our own implementation of caching.
-OPPIA_REDIS_CLIENT = redis.StrictRedis(
+# Here we use MyPy ignore because the redis library's type stubs
```

Comment:
For this and below, could you please point to some reference to explain what exactly is not allowed?

I am wondering whether we can just update our type stubs for redis if needed. But it might help to look at the type stubs for redis to see if we're doing it wrong or if this is a known issue in their stubs.

Reply:
OK, then could you please do the other thing, i.e. point to some reference to explain what exactly is not allowed?